sources/platform/actors/development/programming_interface/environment_variables.md

```diff
@@ -16,59 +16,83 @@ import TabItem from '@theme/TabItem';
-| `ACTOR_STARTED_AT`                 | Date when the Actor was started.                                                                                                                                                                                                                       |
-| `ACTOR_TIMEOUT_AT`                 | Date when the Actor will time out.                                                                                                                                                                                                                     |
-| `APIFY_TOKEN`                      | The API token of the user who started the Actor.                                                                                                                                                                                                       |
-| `APIFY_USER_ID`                    | ID of the user who started the Actor. Note that it might be different than the owner of the Actor.                                                                                                                                                     |
-| `ACTOR_WEB_SERVER_PORT`            | TCP port on which the Actor can start an HTTP server to receive messages from the outside world.                                                                                                                                                       |
-| `ACTOR_WEB_SERVER_URL`             | A unique public URL under which the Actor run web server is accessible from the outside world.                                                                                                                                                         |
+Apify sets several system environment variables for each Actor run. These variables provide essential context and information about the Actor's execution environment.
+
+Here's a table of key system environment variables:
+
+| Environment Variable | Description |
+|----------------------|-------------|
+| `ACTOR_ID` | ID of the Actor. |
+| `ACTOR_RUN_ID` | ID of the Actor run. |
+| `ACTOR_BUILD_ID` | ID of the Actor build used in the run. |
+| `ACTOR_BUILD_NUMBER` | Build number of the Actor build used in the run. |
+| `ACTOR_TASK_ID` | ID of the Actor task. Empty if Actor is run outside of any task, e.g. directly using the API. |
+| `ACTOR_EVENTS_WEBSOCKET_URL` | Websocket URL where Actor may listen for events from Actor platform. |
+| `ACTOR_DEFAULT_DATASET_ID` | ID of the dataset where you can push the data. |
+| `ACTOR_DEFAULT_KEY_VALUE_STORE_ID` | ID of the key-value store where the Actor's input and output data are stored. |
+| `ACTOR_DEFAULT_REQUEST_QUEUE_ID` | ID of the request queue that stores and handles requests that you enqueue. |
+| `ACTOR_INPUT_KEY` | Key of the record in the default key-value store that holds the Actor input. Typically **INPUT**, but may vary. |
```

Comment:
No need to mention the default value here. A link explaining what an input actually is would be helpful.

Reply:
added link to input section of docs